### PR TITLE
[NP-7087] Draggable crunch lab items

### PR DESCRIPTION
**Pull request by KernelDeimos**
This PR is based on [this PR](https://github.com/kgrgreer/foam3/pull/1701), so it's changes will appear here too until that's merged.

Makes items in CrunchLab draggable which can sometimes help with ambiguous renderings.

Bonus - also updated the capability dropdown so Capability Store items appear first.

Minimal updates were made so that arrows are updated when objects are dragged, but they will sometimes go from one object to the next in very illogical ways. I'm going to try to think of a better way to draw arrows before I put more work on this aspect of it.

### CODE REVIEW

File: src/foam/nanos/crunch/menus.jrl [hunk 1]
```diff
@@ -12,6 +12,7 @@ p({
   "class": "foam.nanos.menu.Menu",
   "id": "crunch.crunchlab",
   "label": "Lab",
+  "keywords": ["crunchlab", "crunch lab", "debug crunch", "debug capabilities"],
```
Comment (jlhughes) [inline src/foam/nanos/crunch/menus.jrl]:
```suggestion
  "keywords": ["crunchlab", "crunch lab", "crunch", "lab", "debug crunch", "debug capabilities"],
```

File: src/foam/u2/svg/Position.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/u2/svg/Position.js:7]: header please.

File: src/foam/u2/svg/RelativePosition.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/u2/svg/RelativePosition.js:7]: header please

File: src/foam/nanos/crunch/menus.jrl (hunk 1 above)
Comment (KernelDeimos) [reply src/foam/nanos/crunch/menus.jrl]: These keywords cover that case - typing `crunch` and `lab` alone brings up the menu item

Comment (KernelDeimos) [reply src/foam/nanos/crunch/menus.jrl]: this is what `foam.mlang.predicate.Keyword` does
```javascript
          if ( s.toLowerCase().includes(arg) ) return true;
```

**Pull request merged**